webplugin/js/app/kommunicate-ui.js

```
@@ -1579,8 +1585,16 @@ KommunicateUI = {
                     );
                 } else if (popupTemplateKey === KommunicateConstants.CHAT_POPUP_TEMPLATE.VERTICAL) {
                     var wrapperRect = el.getBoundingClientRect();
+                    var textEl = el.querySelector('.chat-popup-widget-text');
```

Biome (2.3.13)

[error] Shouldn't redeclare 'textEl'. Consider to delete it or rename it.

'textEl' is defined here:


(lint/suspicious/noRedeclare)